recipes/rise/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "rise" %}
+{% set version = "5.2.0" %}
+{% set sha256 = "514eb8784b8d4f7c2af4ba77055fd7a7d8365a87e55eb118d2f7a3ea400dbd7b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
```

Comment:
This is an important change compared with the current situation where I create pkgs for each architecture.
We need extensive testing to know if we don't have problems in any of the architectures we supported.
But I am +1 on this change unless we see associated problems, it simplifies things... and conda 4.3 is old enough by now...

Reply:
Now I'm not sure if `noarch` should be there, since there are different build scripts for each operative systems. Thoughts @conda-forge/core?